### ISSUE TITLE
Uncaught TypeError: this.getBufferRow is not a function

### ISSUE THREAD

**Issue by gexarcha** [ruivieira/latex-friend#13]
[Enter steps to reproduce below:]
The error occurs when I try to sync the PDF

**Atom Version**: 1.7.4
**System**: Ubuntu 14.04.4
**Thrown From**: [latex-friend](https://github.com/ruivieira/latex-friend) package, v0.0.10

### Stack Trace

Uncaught TypeError: this.getBufferRow is not a function

```
At /users/ml/xoex6879/.atom/packages/latex-friend/lib/latex-friend.js:77

TypeError: this.getBufferRow is not a function
    at atom-workspace.syncpdf (/users/ml/xoex6879/.atom/packages/latex-friend/lib/latex-friend.js:77:25)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:260:29)
    at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/usr/share/atom/resources/app.asar/src/window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (/usr/share/atom/resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -6:04.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:32.1.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.normal-mode.power-mode.is-focused)
     -2:31.3.0 vim-mode:activate-insert-mode (atom-text-editor.editor.vim-mode.normal-mode.power-mode.is-focused)
     -2:24.1.0 editor:newline (atom-text-editor.editor.vim-mode.power-mode.is-focused.insert-mode)
     -2:03.5.0 core:save (atom-text-editor.editor.vim-mode.power-mode.is-focused.insert-mode)
     -2:03.1.0 fuzzy-finder:toggle-buffer-finder (atom-text-editor.editor.vim-mode.power-mode.is-focused.insert-mode)
     -2:01.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -1:57.2.0 fuzzy-finder:toggle-buffer-finder (atom-text-editor.editor.vim-mode.power-mode.insert-mode.is-focused)
     -1:56.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  3x -1:36.2.0 latex:build (atom-text-editor.editor.vim-mode.power-mode.insert-mode.is-focused)
     -1:11 application:new-file (ul.list-inline.tab-bar.inset-panel)
     -0:35.3.0 latex-friend:syncpdf (atom-text-editor.editor.vim-mode.power-mode.insert-mode.is-focused)
     -0:23.8.0 latex-friend:showNavigation (atom-text-editor.editor.vim-mode.power-mode.insert-mode.is-focused)
     -0:08.6.0 latex-friend:showNavigationPane (atom-text-editor.editor.vim-mode.power-mode.insert-mode.is-focused)
     -0:00.9.0 latex-friend:syncpdf (atom-text-editor.editor.vim-mode.power-mode.insert-mode.is-focused)
```

### Config

```json
{
  "core": {
    "autoHideMenuBar": true,
    "themes": [
      "atom-dark-ui",
      "atom-light-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
activate-power-mode, v0.5.2
color-picker, v2.1.1
git-diff-details, v1.3.0
highlight-line, v0.11.1
language-julia, v0.6.0
language-latex, v0.6.1
latex, v0.32.0
latex-friend, v0.0.10
latexer, v0.3.0
linter, v1.11.4
merge-conflicts, v1.4.3
minimap, v4.24.3
pigments, v0.29.1
project-manager, v2.9.7
tree-view-git-branch, v0.1.2
tree-view-git-status, v0.2.3
vim-mode, v0.65.1

# Dev
No dev packages
```

**Comment by dpo** [comment]
Same error on OSX.

**Comment by cecetaca** [comment]
Same as @dpo here. Full path "fixes" the issue but error shows up.

**Comment by ruivieira** [comment]
Thanks. It's working on my machine now. Let me know if it's working on yours so I can close the issue.

**Comment by ruivieira** [comment]
Fixed in #13
